jupyter-on-gke/jupyter_config/config-selfauth.yaml

```diff
@@ -12,15 +12,25 @@ hub:
   image: 
     name: <Repo name here>
     tag: latest
-  config: 
+  config:
     JupyterHub:
-      authenticator_class: 'gcpiapjwtauthenticator.GCPIAPAuthenticator'
+      authenticator_class: ${authenticator_class}
     GCPIAPAuthenticator: 
       header_name: 'X-Goog-IAP-JWT-Assertion'
       expected_audience: '/projects/${project_number}/global/backendServices/${service_id}'
+    Authenticator:
```

Comment:
Did we test that these credentials do not work if auth/IAP is enabled, with the current config?

Reply:
Yes, these username/password are not valid Google username/passwords.